docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -813,4 +813,78 @@ Returns/Downloads a `tar` file which contains a `.meta` and a `.log` text
 files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
-Only `GET` is currently supported.
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename
+X-Is-Phantom: bool
+```
+
+Upload a tarred set of DICOM files that is compressed as a `.zip`,`.tar.gz` or 
+`.tgz`. $Filename must be the full name of the fileset and its extension.
+
+$Filename must be named according to the format 
+$PSCID_$CandID_$VisitLabel.ext where ext is one of the extensions above.
```

Comment:
not sure if you'd want to go into this level of detail, but the LORIS-MRI pipeline also accepts filenames with characters trailing the `$PSCID_$CandID_$VisitLabel` convention (e.g., `$PSCID_$CandID_$VisitLabel_part_2.ext`)

the only crucial parts of the filename are that it begins with `$PSCID_$CandID_$VisitLabel` and ends with one of the three file extension types you've already specified

Reply:
@davidblader is there documentation on the MRI side somewhere that can be linked to describing the naming convention instead of duplicating it?

Reply:
@driusan @davidblader @bopucky It is documented in the imaging uploader README under the requirements section: https://github.com/aces/Loris/blob/master/modules/imaging_uploader/README.md#requirements

Reply:
@bopucky the description in this PR should be in line with the current behaviour of the pipeline